frappe/public/js/frappe/form/templates/timeline_message_box.html

```diff
@@ -23,7 +23,7 @@
 					{% } %}
 
 					<div class="text-muted">
-						{{ comment_when(doc.creation) }}
+						{{ comment_when(doc.communication_date) }}
```

Comment:
Comments and other timeline items don't use this, need to still use creation as fallback.